modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -170,8 +170,10 @@ AbaqusUMATStress::computeQpStress()
 
   // Pass through updated stress, total strain, and strain increment arrays
   static const std::array<Real, 6> strain_factor{{1, 1, 1, 2, 2, 2}};
+  // Account for difference in vector order convention: yz, xz, xy (MOOSE)  vs xy, xz, yz
+  // (commercial software)
   static const std::array<std::pair<unsigned int, unsigned int>, 6> component{
-      {{0, 0}, {1, 1}, {2, 2}, {1, 2}, {0, 2}, {0, 1}}};
+      {{0, 0}, {1, 1}, {2, 2}, {0, 1}, {0, 2}, {1, 2}}};
```

Comment:
By "typical use" you mean not standard Voigt notation order?

Reply:
I was trying to refer to the use of the UMAT interface. I guess there can be a variety of not standard Voigt order.